pkg/doc/verifiable/credential_ldp.go

```diff
@@ -0,0 +1,22 @@
+package verifiable
+
+import (
+	"fmt"
+)
+
+// AddLinkedDataProof appends proof to the Verifiable Credential.
+func (vc *Credential) AddLinkedDataProof(context *LinkedDataProofContext) error {
```

Comment:
Is there a reason why the linked data proof is being added after the VC is instantiated?

Reply:
It seems like `AddLinkedDataProof ` actually means create signature and proof AND append the proof.

Reply:
Yes, that is what it means.

Reply:
Yes, that's right. For example, the Issuer wants to issue a new VC. It can be created directly by filling `verifiable.Credential` structure. To make it verifiable, she can use `Credential::AddLinkedDataProof()`. Finally, the credential is marshaled, e.g. to JSON or JWS (this adds another verification layer).